datastore/datastore.go

```diff
@@ -114,7 +113,7 @@ func NewClient(ctx context.Context, projectID string, opts ...option.ClientOptio
 	}
 
 	if projectID == "" {
-		return nil, errors.New("datastore: missing project/dataset id")
+		return nil, fmt.Errorf("datastore: missing project/dataset id")
```

Comment:
In places where no error is being wrapped, you don't need to replace `errors.New()`.

Reply:
@telpirion Thank you for the review. I replaced it for consistency, But I've reverted it now. Please take a look!